Makefile

```diff
@@ -11,12 +11,13 @@ build: clean
 .PHONY: clean
 clean:
 	@rm -rf ./build/*
+	@rm -rf ./man/man[1-9]
 
 # run go test
 # the "-tags daemon" part is temporary
 .PHONY: test
 test:
-	@go test -tags daemon -v $(shell go list ./... | grep -v /vendor/)
+	@go test -tags daemon -v $(shell go list ./... | grep -vE '/vendor/|github.com/docker/cli/man$$')
```

Comment:
Because of missing packages, that are included only after the vendor-on-the-fly part of generate.sh